openjdk-integ-tests/src/test/java/org/conscrypt/javax/net/ssl/SSLSocketTest.java

```diff
@@ -235,6 +235,29 @@ private SecretKey newKey() {
         c.close();
     }
 
+    @Test
+    public void test_SSLSocket_getInputStream_available() throws Exception {
+        TestSSLSocketPair pair = TestSSLSocketPair.create().connect();
+
+        pair.client.getOutputStream().write(new byte[] { 1, 2, 3, 4 });
+        // We read a single byte first because it's okay if available() returns zero
+        // before we've checked the network to see if any packets are available to
+        // be decrypted, but we should show available bytes once we've decrypted a packet
```

Comment:
nit: "to be".  Also below.

Reply:
Sorry, what's the change you want to see here?

Reply:
Oh sorry... Laptop screen wrap, somehow I missed the word "to".  No change required.